WcaOnRails/app/models/user.rb

```diff
@@ -603,6 +604,9 @@ def senior_delegate?
     delegate_status == "senior_delegate"
   end
 
+  def staff_or_any_delegate?
+    staff? || trainee_delegate?
```

Comment:
missing an `end` here. Maybe this is what confuses your RuboCop in the first place? (If the syntax parsing breaks here, the error might manifest several 10s or 100s lines downwards...)

Reply:
Oops yaa sorry for that... I'll fix that. Thank you.

Reply:
And yes, that was the reason what confused RuboCop, even that worked well now.